examples/resources/bloxone_ipam_ha_group/resource.tf

```diff
@@ -0,0 +1,36 @@
+data "bloxone_ipam_dhcp_hosts" "example_host_1" {
+  filter = {
+    name = "TF_TEST_HOST_01"
+  }
+}
+
+data "bloxone_ipam_dhcp_hosts" "example_host_2" {
+  filter = {
+    name = "TF_TEST_HOST_02"
+  }
+}
+
+resource "bloxone_ipam_ha_group" "example" {
+  name = "example_ha_group"
+}
+
```

Comment:
can you create an HA with no hosts, I thought the hosts are mandatory.

Reply:
You are correct, I should remove this